src/components/tooltip.tsx

```diff
@@ -0,0 +1,35 @@
+import clsx from 'clsx'
+import { ReactNode, useState } from 'react'
+
+export default function ToolTip(props: {
+  children: ReactNode
+  className?: string
+  text: ReactNode
+  position?: 'TOP' | 'BOTTOM' | 'LEFT' | 'RIGHT'
+}) {
+  const { position = 'TOP', className = '' } = props
+
+  const [status, show] = useState(false)
+
+  const style = clsx(
+    'absolute bg-black/50 backdrop-blur rounded text-white transition px-2 py-1 z-50 w-fit text-center block',
```

Comment:
Prefer `slate-grey` for the background to match the theme

Reply:
The cards are already stale-grey, so using it would make it the same as existing background